src/main/java/org/hmhb/program/DefaultProgramService.java

```diff
@@ -69,16 +74,169 @@ public Program getById(
     ) {
         LOGGER.debug("getByIds called: ids={}", ids);
         requireNonNull(ids, "ids cannot be null");
-        return dao.findByIdIn(ids);
+        return dao.findAll(ids);
```

Comment:
It turns out that JpaRepository already had a bulk lookup method, so I took out our findByIdIn and used that.